### DISCUSSION TITLE
INTEGRATION

### DISCUSSION THREAD

**Discussion by sabbaperveen** [AlbertSolomon/tubepy#116] [Q&A] [answered]
I WANT TO INTEGRATE THIS TOOL IN MY DJANGO PROJECT . CAN I DO THIS?

**Comment by AlbertSolomon** [comment, marked as answer]
@sabbaperveen I dont think tkinter apps can run directly in a web browser. Their might be a way around it but when i was developing the app, I only had desktop environments in mind.
here some of the resources to help you in your research.
- https://stackoverflow.com/questions/64170043/django-call-a-tkinter-program-using-a-button
- https://www.reddit.com/r/learnpython/comments/apds22/how_can_i_convert_my_tkinter_gui_application_into/
- https://www.quora.com/How-can-I-create-a-desktop-web-based-app-with-Tkinter-and-Django-Flask-Is-it-possible
- https://www.youtube.com/watch?v=DsZV2ltp43A
> FEEL FREE TO ASK CHATGPT LOL! 😅
